pkg/tlsutils/tlsconfig.go

```diff
@@ -75,7 +75,13 @@ func loadRoots(caPath string) (*x509.CertPool, error) {
 		return nil, nil
 	}
 
-	roots := x509.NewCertPool()
+	roots := new(x509.CertPool)
+	// Start with the system cert pool and then add the custom CA cert
+	roots, _ = x509.SystemCertPool()
```

Comment:
why would we need a system cert pool?

Reply:
The convention generally I feel is to start from the system pool and then append custom CAs.. I feel there is no harm in doing this?

Reply:
Hope that helps to make a decision
- https://github.com/kubernetes-sigs/external-dns/pull/5097#issuecomment-2661100102
- https://github.com/kubernetes-sigs/external-dns/pull/5097#issuecomment-2661100667